common.sh

```diff
@@ -194,6 +194,14 @@ else
     MODULE_PATH=$XCCDF_VALUE_MODULE_PATH
 fi
 
+#
+# Module-generated files storage path
+#
+# This should be used as root for things like hooks and
+# attachments.  Don't save files directly here, create
+# a subdirectory for each type of file.
+#
+_MODULE_STORAGE=$VALUE_TMP_PREUPGRADE/storage/$MODULE_PATH
```

Comment:
The directory has to be packed into the tarball. See `preupgrade_dirs` inside `setting.py`.

Reply:
Added.